packages/commands/src/bitcoin/memo/call.ts

```diff
@@ -0,0 +1,93 @@
+import { Command, Option } from "commander";
+import { z } from "zod";
+
+import { memoCallOptionsSchema } from "../../../../../types/bitcoin.types";
+import {
+  addCommonOptions,
+  broadcastBtcTransaction,
+  displayAndConfirmMemoTransaction,
+  fetchUtxos,
+  setupBitcoinKeyPair,
+} from "../../../../../utils/bitcoin.command.helpers";
+import {
+  bitcoinMakeTransactionWithMemo,
+  getDepositFee,
+} from "../../../../../utils/bitcoinMemo.helpers";
+import { validateAndParseSchema } from "../../../../../utils/validateAndParseSchema";
+
+type CallOptions = z.infer<typeof memoCallOptionsSchema>;
+
+/**
+ * Main function that executes the call operation.
+ * Creates and broadcasts both commit and reveal transactions to perform a cross-chain call.
+ *
+ * @param options - Command options including amounts, addresses, and contract parameters
+ */
+const main = async (options: CallOptions) => {
+  const { key, address } = setupBitcoinKeyPair(
+    options.privateKey,
+    options.name
+  );
+  const utxos = await fetchUtxos(address, options.bitcoinApi);
+
+  const data = options.data?.startsWith("0x")
+    ? options.data.slice(2)
+    : options.data;
+
+  const receiver = options.receiver?.startsWith("0x")
+    ? options.receiver.slice(2)
+    : options.receiver;
```

Comment:
I've seen this logic repeated in a few different places, we should definitely create a very simple reusable helper.

Reply:
Now part of a helper:
```ts
const memo = constructMemo(options.receiver, options.data);
```